src/main/java/com/thebois/persistence/LoadSystem.java

```diff
@@ -0,0 +1,50 @@
+package com.thebois.persistence;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+
+/**
+ * Handles loading of saved objects in the save-file.
+ */
+public class LoadSystem extends AbstractPersistence {
+
+    private final ObjectInputStream objectInputStream;
+
+    /**
+     * Creates an instance of an input stream for the save-file.
+     *
+     * @throws IOException If the save-file is deleted during runtime.
+     */
+    public LoadSystem() throws IOException {
+
+        directoryExist();
+        final FileInputStream fileInputStream = new FileInputStream(getSaveFilePath());
+
+        objectInputStream = new ObjectInputStream(fileInputStream);
+    }
+
+    /**
+     * Reads an object from the save-file.
+     *
+     * @return The read object.
+     *
+     * @throws IOException            If the file is not found. Will only occur if file is deleted
+     *                                during runtime.
+     * @throws ClassNotFoundException If there are no objects in the save-file left.
+     */
+    public Object read() throws IOException, ClassNotFoundException {
```

Comment:
As I mentioned previously, it's really ugly to return `Object` straight-up. Even though it is used internally, I believe it would be better to handle the casting of `Object` into `Colony` and `World` internally rather than force the client to cast them.